src/languages/morfeusz_support.py

```diff
@@ -82,6 +139,370 @@ def get_gender_from_token(token: Token):
 
         return MORFEUSZ_GENDER_TYPES[morfeusz_gender]
 
+    variants = {
```

Comment:
Maybe it would be better to put the content of `variants` in a separate json file? 
Then this line would just be `variants = json.load(filename)` which would be easier to read and maintain?

Reply:
Sorry, I missed your comments. I just committed and pushed the version with polish variants moved to a separate json file.

Reply:
Thanks! Merging now.